### PR TITLE
WIP - Improve coverage on utils and exporters

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

File: src/services/Api/ActionBuilder.ts [hunk 1]
```diff
@@ -60,15 +60,12 @@ export class ActionBuilder {
     protected buildQueryString() {
         const parsedURL = new URL(this.getUrl(), 'http://dummybase');
         const querySeparator = parsedURL.searchParams.toString() === '' ? '?' : '&';
-        const queryString = this.getQueryParams() ? new URLSearchParams(this.stringParams(this.getQueryParams())).toString() : '';
+        const queryParams = this.getQueryParams();
+        const queryString = queryParams ? new URLSearchParams(this.stringParams(queryParams)).toString() : '';
```
Comment (spinder) [inline src/services/Api/ActionBuilder.ts:64]: I think we should explicitly expand all of the chained checks here to 
 i)clarify that all the null checks are happening.  It feels like we lost a check here in the refactor.  It's a little less hard to follow as it's currently written, but further expansion would probably help.  I'm not clear what value queryString is assigned if 'this.getQueryParams()' or one of these constructors fails to instantiate 'URLSearchParams(..)', 'this.stringParams()'.  Feels like a null string maybe.

Comment (josejulio) [reply src/services/Api/ActionBuilder.ts:64]: `this.stringParams()` will always return `Record<string, string>` and it will always expect a valid `QueryParamsType` (is on the function signature, the compiler will enforce that).
I can make it more concise by using a regular `if/else`, is that what you mean?

Comment (josejulio) [reply src/services/Api/ActionBuilder.ts]: Expanded this, please have a look
